dashboard/config/blocks/GamelabJr/gamelab_makeNewSpriteGroup.json

```diff
@@ -0,0 +1,59 @@
+{
+  "category": "",
+  "config": {
+    "func": "makeNewSpriteGroup",
+    "blockText": "make {N} new {GROUP} in a {FORMAT}",
+    "style": "sprite_blocks",
```

Comment:
Thanks for adding styles to the blocks! This will be especially helpful once we migrate Sprite Lab to Google Blockly. In the meantime, because CDO Blockly doesn't support styles like this, we'll want to also add a hard-coded HSV value so that the block gets the expected red color.

You should just need to add this:
```
  "color": [
    355,
    ".7",
    ".7"
  ],
```

(I don't have have these values memorized, so I just checked another sprite block from the same pool.)

Reply:
The layout block should just stay blue, yeah? Red = creates sprites?